website/static/js/_anonymousLogTexts.json

```diff
@@ -0,0 +1 @@
ser unlinked content from Figshare in a project","figshare_file_added":"A user added a file to Figshare in a project","figshare_file_removed":"A user removed a file from Figshare in a project","figshare_node_authorized":"A user authorized the Figshare addon from a project","figshare_node_deauthorized":"A user deauthorized the Figshare addon from a project","forward_url_changed":"A user changed the forward URL in a project","github_file_added":"A user added a file to a GitHub repo in a project","github_file_removed":"A user removed a file in a GitHub repo in a project","github_file_updated":"A user updated a file in a GitHub repo in a project","github_folder_created":"A user created a folder in a GitHub repo in a project","github_node_authorized":"A user authorized the GitHub addon for a project","github_node_deauthorized":"A user deauthorized the GitHub addon for a project","github_repo_linked":"A user linked a GitHub repo in a project","googledrive_file_added":"A user added a file to Google Drive in a project","googledrive_file_removed":"A user removed a file from Google Drive in a project","googledrive_file_updated":"A user updated a file in Google Drive in a project","googledrive_folder_created":"A user created a folder in Google Drive in a project","googledrive_folder_selected":"A user linked a Google Drive folder to a project","googledrive_node_authorized":"A user authorized the Google Drive addon for a project","googledrive_node_deauthorized":"A user deauthorized the Google Drive addon for a project","mendeley_folder_selected":"A user selected a folder in Mendeley in a project","mendeley_node_authorized":"A user authorized the Mendeley addon for a project","mendeley_node_deauthorized":"A user deauthorized the Mendeley addon for a project","permissions_updated":"A user changed permissions for a project","s3_bucket_linked":"A user linked an Amazon S3 bucket to a project","s3_bucket_unlinked":"A user unselected an Amazon S3 bucket in a project","s3_file_added":"A user added a file to an Amazon S3 bucket in a project","s3_file_removed":"A user removed a file in an Amazon S3 bucket in a project","s3_file_updated":"A user updated a file in an Amazon S3 bucket in a project","s3_folder_created":"A user created a folder in an Amazon S3 in a project","s3_node_authorized":"A user authorized the Amazon S3 addon for a project","s3_node_deauthorized":"A user deauthorized the Amazon S3 addon for a project","zotero_folder_selected":"A user linked a Zotero folder to a project","zotero_node_authorized":"A user authorized the Zotero addon for a project","zotero_node_deauthorized":"A user deauthorized the Zotero addon for a project","dataverse_study_released":"A user published a new version of a Dataverse dataset to a project","dataverse_node_deauthorized_no_user":"Dataverse addon for a project deauthorized","affiliated_institution_added":"A user added an affiliation to a project","affiliated_institution_removed":"A user removed an affiliation from a project"}
```

Comment:
We shouldn't need to commit both this and the `.js` version. `_allLogTexts.json` is .gitignored, this probably should be too.